gnovm/stdlibs/native.go

```diff
@@ -334,6 +334,31 @@ var nativeFuncs = [...]nativeFunc{
 				p0, p1, p2, p3)
 		},
 	},
+	{
+		"std",
+		"emitEvent",
+		[]gno.FieldTypeExpr{
+			{Name: gno.N("p0"), Type: gno.X("string")},
+			{Name: gno.N("p1"), Type: gno.X("[]string")},
```

Comment:
BTW, gno.X("...string") would make this variadic, like gno.Vrd(gno.X("string")). 
Variadic might be more efficient, any reason not to use it?

Reply:
well actually I tried to declare `attrs` as a `...string` type, but since ellipsis is not currently implemented in code generation when injecting the go code into gno. So, I set it as a `[]string` type instead.

Anyway, this part needs to be fixed if it is supported in the future.

https://github.com/gnolang/gno/blob/3d1d26cbba76b4ad1fd5d13dc561bf1958f9994e/misc/genstd/mapping.go#L235-L243

cc: @thehowl